pkgs/top-level/all-packages.nix

```diff
@@ -22068,6 +22068,57 @@ with pkgs;
                 );
     }).config.system.build;
 
+
+  /*
+   * Run a NixOS VM network test.
+   *
+   * This is equivalent to  import ./make-test.nix  from the NixOS manual
```

Comment:
Nitpick:
```
`import ./make-test.nix`
```
instead of two spaces to keep consistency.

Reply:
We don't really have a syntax for comments, but why not :)